gcloud/bigtable/__init__.py

```diff
@@ -16,3 +16,11 @@
 
 
 from gcloud.bigtable.client import Client
+
+try:
+    import grpc.beta.implementations
+except ImportError as exc:
+    message = ('gRPC is required for using the Cloud Bigtable API. '
```

Comment:
Can we do some Python version checking here?

For Python 3+:

> gRPC is required for using the Cloud Bigtable API, but importing the gRPC library (grpcio on PyPI) failed. Currently, grpcio is only supported in Python 2.7, which unfortunately means the Cloud Bigtable API isn't available.

For Python 2.7 with an import error

> gRPC is required for using the Cloud Bigtable API, but importing the gRPC library (grpcio on PyPI) failed. This likely means you have a non-standard version of Python installed. Check http://grpc.io if you're having trouble installing the gRPC package.

Or just combine them all...

> gRPC is required for using the Cloud Bigtable API, but importing the gRPC library (grpcio on PyPI) failed.
>
> Currently grpcio is only supported in Python 2.7, which unfortunately means the Cloud Bigtable API isn't available if you're using Python 3. If you're using Python 2.7, this likely means you have a non-standard version of Python installed. Check http://grpc.io if you're having trouble installing the gRPC package.